### PR TITLE
Misc changes for first integration.

### PR DESCRIPTION
**Pull request by nhsd-jonathan-heather**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/azure-release-pipeline.yml [hunk 1]
```diff
@@ -68,7 +68,5 @@ extends:
       short_service_name: "${{ variables.short_service_name }}-sa"
       product_display_name: "${{ variables.product_display_name }} (ERSSA)"
       service_base_path: "${{ variables.service_base_path }}-sa"
-      jinja_templates:
-        ERS_TARGET_SERVER: e-referrals-service-api--sa
```
Comment (nhsd-jack-wainwright) [inline azure/azure-release-pipeline.yml:72]: Is this no longer required? How do we now define the proxy name for each environment?

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me :)

File: azure/azure-release-pipeline.yml (hunk 1 above)
Comment (nhsd-jonathan-heather) [reply azure/azure-release-pipeline.yml:72]: It is no longer required after https://github.com/NHSDigital/api-management-infrastructure/pull/302/files has been merged and deployed.

It was wrong to have an explicitly named target server for SA (it should be just e-referrals-service-api) for any Apigee environment that only has one e-RS environment mapped.

Another way to put it: only Apigee's "internal-dev" has >1 e-RS environments where we need to distinguish between different target servers.

**Review by nhsd-hubert-raich — APPROVED** [review]

**Pull request merged**